apps/client/src/routes/dashboards/dashboard/dashboard-page.tsx

```diff
@@ -30,11 +30,13 @@ export function DashboardPage() {
     return <DashboardLoadingState />;
   }
 
+  const awsRegion = Auth.configure().region || 'us-west-2';
```

Comment:
Nit: It's more semantically correct to use `??` instead of short circuiting these days.

Reply:
I noticed this from the linter and double checked that since `Auth.configure().region` is a string the logic should work, but I probably should have just taken the suggestion and changed it to `??`.